nixos/modules/i18n/input-method/fcitx5.nix

```diff
@@ -32,6 +32,7 @@ in {
         GTK_IM_MODULE = "fcitx";
         QT_IM_MODULE = "fcitx";
         XMODIFIERS = "@im=fcitx";
+        QT_PLUGIN_PATH = [ "${fcitx5Package}/${pkgs.qt6.qtbase.qtPluginPrefix}" ];
```

Comment:
does that interfere with qt5 apps ?

Reply:
Qt automatically rejects incompatible plugins, thus nothing to worry about.